chapter-06/recipe-02/fortran-c-example/configurator.py

```diff
@@ -0,0 +1,10 @@
+def configure_file(input_file, output_file, vars_dict):
```

Comment:
I like the fact that preparing the configuration dictionary has been factored out. Indeed it makes slightly more sense like this in the original context for which this recipe was thought.